test/framework/src/main/java/org/opensearch/test/OpenSearchIntegTestCase.java

```diff
@@ -270,6 +270,12 @@ public abstract class OpenSearchIntegTestCase extends OpenSearchTestCase {
      * Property that controls whether ThirdParty Integration tests are run (not the default).
      */
     public static final String SYSPROP_THIRDPARTY = "tests.thirdparty";
+    public static final List<String> CODECS = List.of(
```

Comment:
:+1: @sarthakaggarwal97 could you please add the comment why `lucene_default` is not included (so one won't be wondering why it is not there), thank you

Reply:
@reta please check now!